resources/production/chainspec.toml

```diff
@@ -48,24 +48,21 @@ era_duration = '30minutes'
 # Minimum number of blocks per era.  An era will take longer than `era_duration` if that is necessary to reach the
 # minimum height.
 minimum_era_height = 10
-# Amount of time to go back before the start of the era for picking the booking block.
-booking_duration = '0ms'
-# Amount of time to wait after the booking before we pick the key block, collecting the magic bits along the way.
-entropy_duration = '0ms'
-# Keep voting on the switch block for a fixed amount of time; effective if the summit level is zero.
-voting_period_duration = '2days'
-# Integer between 1 and 99 representing the fault tolerance threshold as a percentage, used by the internal finalizer.
-# It is the percentage of validators that would need to equivocate to make two honest nodes see two conflicting
+# A number between 0 and 1 representing the fault tolerance threshold as a fraction, used by the internal finalizer.
+# It is the fraction of validators that would need to equivocate to make two honest nodes see two conflicting
 # blocks as finalized: A higher value F makes it safer to rely on finalized blocks. It also makes it more difficult to
-# finalize blocks, however, and requires strictly more than 50% + F/2 validators to be working correctly.
-finality_threshold_percent = 10
+# finalize blocks, however, and requires strictly more than (F + 1)/2 validators to be working correctly.
+finality_threshold_fraction = [1, 10]
```

Comment:
This should be 1/3 in my opinion… but it looks like it doesn't even have an entry in [FEAT-217](https://casperlabs.atlassian.net/browse/FEAT-217). :confused:

Reply:
1/3? SO it's gonna be `(1/3 + 1)/2`?

Reply:
Which is 2/3, yes. Meaning we can tolerate up to 1/3 validators who are malicious or crashed.

Reply:
I think we changed it because we had too many liveness failures in testnets, no?
In any case, I think maybe changes to configuration values should be a separate PR and more widely discussed? Then this task would be just about changing the form in which the values are expressed.